### DISCUSSION TITLE
Issue while Installing Foundry

### DISCUSSION THREAD

**Discussion by Not-Sarthak** [Cyfrin/foundry-full-course-cu#981] [Q&A]
Hey, I am getting this error while installing Foundry on Windows.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/92942966/55c836c9-e13e-4a7c-90b3-d63641f581b2)

I've read previous discussions which were related to the same issue and have found that Foundry isn't supported on windows. Can anyone help me with what should I do next?

**Comment by 0xusmanf** [comment]
[Here](https://youtu.be/umepbfKp5rI?t=23562) is a step-by-step guide by Vasiliy explaining how to work with Foundry on Windows by installing WSL.

**Reply by Not-Sarthak** [reply]
hey! thanks for responding but I tried following the same yet unable to install WSL because of the same error.

**Reply by 0xusmanf** [reply]
Are you getting this error while installing Foundry or WSL? and can you please specify the command you are using along with the resulting log?

**Reply by theirrationalone** [reply]
If not mitigated yet?? Please [try this out](https://github.com/Cyfrin/foundry-full-course-f23/discussions/977)

**Reply by Not-Sarthak** [reply]
> Are you getting this error while installing Foundry or WSL? and can you please specify the command you are using along with the resulting log?
> 
> 

I was getting the same error while installing WSL but it got resolved now :))

**Reply by Not-Sarthak** [reply]
Yes that helped :) thanks a lot
